modules/@apostrophecms/seo-fields-doc-type/index.js

```diff
@@ -42,6 +44,42 @@ module.exports = {
           }
         }
       };
+
+      if (self.options.seoCanonicalTypes
+        && Array.isArray(self.options.seoCanonicalTypes)
```

Comment:
Lint errors: `&& should be places at the end of the line`
[Optional] for readability:
 ```
       if (
        self.options.seoCanonicalTypes &&
        Array.isArray(self.options.seoCanonicalTypes) &&
        self.options.seoCanonicalTypes.length
      ) {
      ```
      + could early return here too.